pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -44,7 +44,7 @@ include::modules/ref_pantheon-user-interface.adoc[leveloffset=+2]
 
 include::modules/proc_log-in-to-pantheon.adoc[leveloffset=+3]
 
-// include::<pending>
+include::assembly-searching-modules.adoc[leveloffset=+2]
```

Comment:
I don't think we support nested assemblies in pantheon v2.
@xdavidson can you please confirm?